WKElementaryDark.css

```diff
@@ -210,6 +210,50 @@
     --EDI-footer-filter: invert(1);
     --EDI-loading-filter: grayscale(90%) invert(1) hue-rotate(180deg);
     --EDI-overlay-box-highlights-filter: hue-rotate(160deg);
+
+    /* 
+     * Redefine Wanikani provided properties to use ED props
+     */
+    --color-black: var(--EDI-pure-black);
+    --color-gray-light: var(--EDI-gray-12);
+    --color-white: var(--EDI-pure-white);
+    --color-red: var(--EDI-red);
+    --color-text: var(--ED-text);
+    --color-radical: var(--ED-radical);
+    /* --color-radical-gradient: $colorRadicalBlueGradient; */
+    --color-kanji: var(--ED-kanji);
+    /* --color-kanji-gradient: $colorKanjiPinkGradient; */
+    --color-vocabulary: var(--ED-vocab);
+    /* --color-vocabulary-gradient: $colorvocabularyPurpleGradient; */
+    --color-subject-slide-navigation-background: var(--ED-surface-2);
+    --color-subject-slide-navigation-text: var(--ED-text);
+    --color-subject-slide-navigation-button-hover: var(--ED-text-hl);
+    --color-button-default-background: var(--ED-surface-4);
+    --color-button-quiz-background: var(--ED-success);
+    --color-button-quiz-text: var(--ED-text);
+    --color-button-quiz-text-shadow: none;
+    --color-button-modal-primary-background: var(--ED-surface-4);
+    --color-button-modal-primary-border: var(--color-white);
+    --color-button-modal-primary-text: var(--ED-text);
+    --color-button-modal-primary-text-shadow: none;
+    --color-button-modal-secondary-background: transparent;
+    --color-button-modal-secondary-border: var(--color-white);
+    --color-button-modal-secondary-text: var(--ED-text);
+    --color-button-modal-secondary-text-shadow: none;
+    /* --color-text-shadow-dark: rgba(0, 0, 0, 0.2); */
+    --color-text-shadow-light: none;
+    --color-quiz-incorrect-background: var(--ED-incorrect);
+    --color-quiz-incorrect-text-color: var(--ED-text);
+    --color-quiz-incorrect-text-shadow: none;
+    --color-quiz-correct-background: var(--ED-success);
+    --color-quiz-correct-text-color: var(--ED-text);
+    --color-quiz-correct-text-shadow: none;
+    --color-quiz-srs-correct-background: var(--ED-success);
+    --color-quiz-srs-correct-text-color: var(--color-white);
+    --color-quiz-srs-correct-text-shadow: none;
+    --color-quiz-srs-incorrect-background: var(--ED-incorrect);
+    --color-quiz-srs-incorrect-text-color: var(--ED-text);
+    --color-quiz-srs-incorrect-text-shadow: none;
```

Comment:
This seems dangerous. The amount of testing required to make sure this doesn't result in any unwanted behaviour is making my head spin.